io.github.kaschpal.timetable.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "io.github.kaschpal.timetable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
```

Comment:
Does it really needs the GNOME SDK?

Reply:
I tried it locally without the sdk line. At least building with Gnome Builder fails because of "sdk not specified".

Reply:
What I meant is using org.freedesktop.Sdk

Reply:
Switching both runtime and sdk to freedesktop does not work at an "import gi".
I tried to leave the runtime at gnome and change the sdk to freedesktop, but there does not seem to be an option "sdk-version". So the sdk complains, there is no version 3.36. (freedesktop should be 19.08, I think)